specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/ManagedDatabaseSecurityAlertPolicies.json

```diff
@@ -159,6 +159,59 @@
           }
         }
       }
+    },
```

Comment:
any reason for not supporting PATCH   ? (for ex to allow email info to be updated )

Reply:
The policy is updated through PUT, so PATCH is not necessary.

Reply:
PATCH is not strictly required for proxy resources.